### PR TITLE
Migrated documentation of HipChat to WunderWay

### PR DESCRIPTION
**Pull request by huberfl**
This PR resolves issue #65

### CODE REVIEW

**Comment by huberfl** [comment]
Once this PR is merged, we can remove the doc from Drive: https://docs.google.com/document/d/18R-RglYUJuLOYT-D2LxbtxkTLQNBlNB1Vf91SwbLA8c/edit#

@steveparks, can you please keep an eye on this? Or shall I just ping you?

**Comment by steveparks** [comment]
I'll try to keep an eye - but it's worth you or whoever merges this pinging me as a nudge.

**Comment by steveparks** [comment]
Great idea to merge this in @huberfl - I've reviewed and made a few minor comments.

File: working-at-wunderkraut/communicating/index.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ title: Communicating
 ---
 ## Individuals and interactions over processes and tools
 As a rule of thumb, we adhere to the [Manifesto for Agile Software Development](http://www.agilemanifesto.org), which states - besides others - “Individuals and interactions over processes and tools”. 
+In most of our projects, we have to consider the highly remote setup. Within these circumstances, interactions between individuals are not always to be considered (and efficient) to be face-to-face. According to this, in order to discuss issues and reach mutual agreement, we prefer the following communication channels in their given order:
```
Comment (huberfl) [inline working-at-wunderkraut/communicating/index.md]: What about moving this paragraph to the page "communication-channels" and just provide a reference here?

**Comment by joesb** [comment]
Looks good to me. Thanks for adding this to WunderWay @huberfl

**Pull request merged**